tests/integration/settings.py

```diff
@@ -0,0 +1,7 @@
+import os
+
+# URL for cartography integration tests
+NEO4J_URL = os.environ["NEO4J_URL"] if "NEO4J_URL" in os.environ else "bolt://localhost:7687"
```

Comment:
`.get()` can be used on the `os.environ` object to specify a default in a more elegant manner.